airbyte-integrations/connectors/source-mysql/src/test-integration/java/io/airbyte/integrations/source/mysql/AbstractMySqlSourceDatatypeTest.java

```diff
@@ -412,6 +400,26 @@ protected void initTests() {
 
   }
 
+  protected void addJsonDataTypeTest() {
+    addDataTypeTestData(
+        TestDataHolder.builder()
+            .sourceType("json")
+            .airbyteType(JsonSchemaType.STRING)
+            .addInsertValues("null", "'{\"a\": 10, \"b\": 15}'", "'{\"fóo\": \"bär\"}'", "'{\"春江潮水连海平\": \"海上明月共潮生\"}'")
+            .addExpectedValues(null, "{\"a\": 10, \"b\": 15}", "{\"fóo\": \"bär\"}", "{\"春江潮水连海平\": \"海上明月共潮生\"}")
+            .build());
+  }
+
+  protected void addTimestampDataTypeTest() {
+    addDataTypeTestData(
+        TestDataHolder.builder()
+            .sourceType("timestamp")
+            .airbyteType(JsonSchemaType.STRING_TIMESTAMP_WITH_TIMEZONE)
+            .addInsertValues("null", "'2021-01-00'", "'2021-00-00'", "'0000-00-00'", "'2022-08-09T10:17:16.161342Z'")
+            .addExpectedValues(null, null, null, null, "2022-08-09T10:17:16.000000Z")
```

Comment:
why are these returning null? is it because the insert values aren't valid timestamps?

Reply:
Yup